### DISCUSSION TITLE
#Beginner's guide

### DISCUSSION THREAD

**Discussion by DuajDiaz** [DuajDiaz/Travel-Website#21] [Q&A] [answered]
How to learn programming?

**Comment by Bunny77K9** [comment, marked as answer]
Learn. Code. Practice in an Infinite Loop! You use Coursera, LinkedinLearn, Udemy, Udacity, Sololearn and many other free learning platforms to learn different kinds of programming languages. It's not enough that u have watched it. If you need to become a pro in that language, you need to practice on a daily basis.

**Reply by DuajDiaz** [reply]
Thank for your answer!
